packages/skeleton-svelte/src/lib/components/Rating/RatingItem.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
```

Comment:
I think there's a case to be made against dropping the child `<RatingsItem>` component. While I know Ratings can have multiple children - they're basically always clones of the same few combinations of icon states (ex: all stars).

I think we can follow the prior Skeleton iteration and keep a `max` prop, which generates the X iterations within a loop. Much like [Zag's own doc example](https://zagjs.com/components/react/rating-group#usage):

- https://github.com/skeletonlabs/skeleton/blob/next/packages/skeleton-svelte/src/lib/components/Rating/Rating.svelte#L6
- https://github.com/skeletonlabs/skeleton/blob/next/packages/skeleton-svelte/src/lib/components/Rating/Rating.svelte#L93

The edge case of someone wanting to implement custom icons per each and every child seems highly unlikely. And that's all the child components really buy us here.

TL;DR: K.I.S.S.